lib/test/src/logger_test.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ *  Distributed under the MIT License (See accompanying file /LICENSE )
+ */
```

Comment:
Is this under MIT license? If so, the link to the file /LICENSE is wrong.

Reply:
damned copy & paste...